DOCS/man/options.rst

```diff
@@ -7111,6 +7111,16 @@ Miscellaneous
 
     This is a key/value list option. See `List Options`_ for details.
 
+``--backdrop-type=<auto|none|mica|acrylic|mica-alt>``
+    (Windows only)
+	Controls the backdrop/border style.
```

Comment:
indent is wrong here.

Reply:
Hmm, surely there are some linting solutions that can check tab/space correctness?
easy to miss otherwise